### PR TITLE
xontrib-ssh-agent

### PR DESCRIPTION
**Pull request by scopatz**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

noarch: python

### CODE REVIEW

**Comment by scopatz** [comment]
@conda-forge/staged-recipes - this is a `noarch: python` recipe, and so is ready for review / merge

File: recipes/xontrib-ssh-agent/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+test:
+  commands:
+    - test -f "${SP_DIR}/xontrib/ssh_agent.xsh"
+
+about:
+  home: https://bitbucket.org/dyuri/xontrib-ssh-agent
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```
Comment (jakirkham) [inline recipes/xontrib-ssh-agent/meta.yaml]: Would be good to ask upstream to ship this in the PyPI package.

**Comment by jakirkham** [comment]
Thanks @scopatz! 😄

File: recipes/xontrib-ssh-agent/meta.yaml (hunk 1 above)
Comment (scopatz) [reply recipes/xontrib-ssh-agent/meta.yaml]: yep!

**Pull request merged**